### ISSUE TITLE
Site Support Request: http://www.schooltube.com/ (SchoolTube)

### ISSUE THREAD

**Issue by wertercatt** [rg3/youtube-dl#13091]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2017.05.14*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2017.05.14**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [x] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: http://www.schooltube.com/video/dc30fcfe08864571af1f/Half-Life%20Snacks
- Single video: http://www.schooltube.com/video/dc30fcfe08864571af1f/
- Organization (like a YouTube channel's /videos link): http://www.schooltube.com/organization/211565/
- Channel page (seems to also be a page with its own video as well?): http://www.schooltube.com/channel/vvhsvocskills/

Note that **youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free)**. In order for site support request to be accepted all provided example URLs should not violate any copyrights.

---

### Description of your *issue*, suggested solution and other information

Explanation of your *issue* in arbitrary form goes here. Please make sure the [description is worded well enough to be understood](https://github.com/rg3/youtube-dl#is-the-description-of-the-issue-itself-sufficient). Provide as much context and examples as possible.
If work on your *issue* requires account credentials please provide them or explain how one can obtain them.

Metadata extractable from single video page:
- The title can be extracted from the "og:title" meta tag.
Example: `<meta property="og:title" content="Half-Life Snacks"/>`
- The description can be extracted from the "description" or "og:description" meta tags.
Example: `<meta name="description" content="Award of Excellence at the 2009 Northwest High School Film Festival: Documentary Category
"/>`
Example 2: `<meta property="og:description" content="Award of Excellence at the 2009 Northwest High School Film Festival: Documentary Category
"/>`
- The tags can be extracted from the "keywords" meta tag.
Example: `<meta name="keywords" content="Northwest  High  School  Film  Festival">`
- The view count, upload date, upload user, and organization can be extracted from the "video-meta-data" HTML class
Example:
```html
                    <div class="video-meta-data">
                        <div class="view-count">4,826 views</div>
                        <div>Uploaded on Aug 15, 2009</div>
                        <div>by nwhsff (Teacher)</div>
                        
                            <div>
                                <a href="/organization/211565/">
                                    Mount Si High School
                                </a>
                            </div>
                        
                    </div>
```
- The upload date can also be retrieved from the Last-Modifed headers of the video file or thumbnail. (Which the generic extractor is able to locate within the page currently.)

**Comment by dkl3** [comment]
The generic extractor works in the meantime:

youtube-dl "http://www.schooltube.com/video/ae65d211000a45d292fe/13.%20Pringle's%20Rocket%20-%20Ethan"

Put url in quotes.

**Comment by wertercatt** [comment]
Yeah, but I'd like for it to be able to extract the metadata as well.
